src/app.py

```diff
@@ -31,7 +34,7 @@ def setup_players(self):
         tokens = self.cli.get_player_tokens()
         builder = PlayerBuilder(self.validator, self.processor, self.cli)
         players = builder.build_players(tokens)
-
+        if players == constants.EXIT: self.exit_app()
```

Comment:
I understand what's happening here, but in terms of code transparency, it seems strange to me to compare something called "players" to something called "EXIT". "EXIT" doesn't seem to me to be a state that a player could have, so why check if it has that state?